frontend/src/pages/ApplicationsPage.tsx

```diff
@@ -18,7 +18,7 @@ import {
 } from '@patternfly/react-core';
 
 type ApplicationsPageProps = {
-  title: React.ReactNode;
+  title?: React.ReactNode;
```

Comment:
The usage of `GlobalPipelineCoreLoader` which uses `ApplicationsPage` does not require a title.